.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report a bug, false-positive or false-negative
+title: '[BUG] '
+labels: ''
```

Comment:
Do you want to have some labels applied automatically?

Reply:
For bug reports, probably not. The only truly unambiguous case against our triage is likely to be the 'new rule' template.

Reply:
I couldn't find any existing `new-rule` label.

Reply:
No label sounds good here 👍 

If we add a label for this, we can update it.